inject-java/src/main/java/io/micronaut/annotation/processing/visitor/JavaClassElement.java

```diff
@@ -618,8 +622,8 @@ public Optional<ClassElement> getEnclosingType() {
             Element enclosingElement = this.classElement.getEnclosingElement();
             if (enclosingElement instanceof TypeElement typeElement) {
                 return Optional.of(visitorContext.getElementFactory().newClassElement(
-                        typeElement,
-                        elementAnnotationMetadataFactory
```

Comment:
you have 2 styles in these classes. Which `continuation indent` right? 8 spaces or 4 spaces?

Reply:
@graemerocher could you tell me which codestyle is right?